Description: Add package git repository files
 Adding the files associated with this debain package's git repository.
 .
 python-ntlm (1.0.1-1) stable; urgency=low
 .
   * Initial release.
Author: Orvant Maintainers <dev@orvant.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ python-ntlm-1.0.1/README.md
@@ -0,0 +1,4 @@
+debian-python-ntlm
+==================
+
+Debian python-ntlm package
